release-notes.txt

```diff
@@ -1,3 +1,751 @@
++ testbed/iotlab: add mapping for samr34-xpro and nucleo-wl55jc (#17578)
++ tests/gnrc_rpl: add automated test for gnrc_rpl (#17353)
++ tools/pr_check: add "Update" keywork to "needs squashing" check (#17822)
+* make/emulate: use unix sockets with qemu (#17679)
+* Makefile.include: pass IOTLAB_NODE to docker (#17545)
+* makefiles/docker.ink.mk: do not always pass CFLAGS to docker (#17818)
+* makefiles/libc/newlibc: allow toolchains with nano version only (#17553)
+* makefiles: socat set tty mode 8N1 (#17853)
+* makefiles: use C++14 standard by default for C++ compilations (#17479)
+* tools/dhcpv6-pd_ia: create /run/kea on startup (#17674)
+
+Kconfig
+-------
+
++ boards/Kconfig: remove wrongfully added MODULE_AUTO_INIT_LED* sym (#17684)
++ cpu/stm32: Add clock config for mp1 to kconfig (#17521)
+* board/*/kconfig: Fix boards on kconfig blocklist (#17467)
+* boards: Use BOARD.config pattern for kconfig (#17648)
+* cpu/stm32/wl: Model kconfig clocks (#17496)
+* drivers/*/Kconfig: Cleanup of simple drivers (#17669)
+* pkg/littlefs2: model Kconfig (#17882)
+* sys/chunked_ringbuffer: model Kconfig (#17744)
+* sys/fido2: model Kconfig (#17435)
+* {disp,touch}_dev: improve Kconfig at different levels (#17458)
+
+Examples
+--------
+
++ examples/gcoap_block_server: add gcoap block server example (#17843)
++ examples/lorawan: add the possibility to use ABP activation procedure (#11237)
+* examples/gcoap_dtls: match Makefile of examples/gcoap (#17552)
+* examples/twr_aloha: blacklist test on ci (#17557)
+* Make example/emcute_mqtt honor radio settings (#17773)
+
+Testing
+-------
+
++ .github/workflows/test-on-iotlab: add dwm1001 (#17558)
++ tests/*: add BOARDs to Makefile.ci (#17754)
++ tests: add default BOARDS (#17715)
+* CI: disable microbit testing [backport 2022.04] (#17953)
+* murdock: prioritize job collection over build jobs (#17610)
+* murdock: support emulated boards (#17434)
+* murdock: use json output `$(BINDIR)` size (#17213)
+* tests/*: remove uneeded xtimer_init (#17566)
+* tests/*xtimer*: remove uneeded timex calls (#17728)
+* tests/driver_dfplayer: use event_thread instead of
+  event_thread_lowest (#17750)
+* tests/periph_uart_nonblocking: migrate to ztimer (#17567)
+* tests/pkg_edhoc: handle multiple interfaces (#17534)
+* tests/pkg_relic: increase stacksize (#17528)
+* tests/senml: do no include saul_default (#17641)
+* tests/sys_atomic_utils: use ztimer_usec (#17851)
+* tests/sys_crypto: fix potentially unitialized error (#17718)
+* tests/thread_float: use ztimer_usec (#17835)
+* tests: multiple fixes (#17840)
+* workflow/tool-test:  update prior install (#17738)
+
+API Changes
+-----------
+
++ drivers/periph_common: add periph_init_buttons to init on-board
+  buttons (#17711)
+* drivers/atwinc15x0: register with netdev (#17887)
+* sys/pm_layered: use array representation, get rid of implicit IDLE
+  mode (#17895)
+* tests: rework eth drivers (#17813)
+* ztimer_periodic: make callback function return bool (#17351)
```

Comment:
better point out that with the old API `ZTIMER_PERIODIC_KEEP_GOING` was 0, now it's `true` - so the logic is reversed now.

For users of the macro, nothing changes.